frontend/sozisel-app/src/hooks/useFetchEventLiveResult/useFetchPollLiveResult.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  EventResultSubmittedSubscription,
+  PollOptionSummary,
+  usePollSummaryQuery,
+} from "../../graphql";
+import { useEffect, useState } from "react";
+import { Event } from "../../model/Template";
+
+type PollOption = Pick<PollOptionSummary, "id" | "text" | "votes">;
+
+export interface PollSummary {
+  optionsSummary: PollOption[];
+  totalVoters: number;
+  mostAnsweredOption: string;
+}
+
+interface UseFetchPollLiveResultProps {
+  eventResult?: EventResultSubmittedSubscription;
+  skip: boolean;
+  event: Event;
+  eventId: string;
+}
+
+//sorting options in descending order
+const sortOptions = (options: PollOption[]): PollOption[] =>
+  options.slice().sort((a, b) => b.votes - a.votes);
+
+//if there are more than one option with the same number of votes concant options text of these options e.g. Nie, Tak
+const getMostAnsweredOptionsText = (sortedOptions: PollOption[]): string =>
+  sortedOptions
+    ?.filter((option) => option.votes === sortedOptions[0].votes)
```

Comment:
do you need the `?.`?

Reply:
if so then argument declaration is wrong

Reply:
Yeah, I added `?` during testing. Now it is unnecessary.